common/changes/@microsoft/api-extractor/josmithr-api-extractor-fix-config-inheritence_2025-01-09-00-41.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Update merge behavior for derived configurations to allow overriding array properties",
+      "type": "minor"
```

Comment:
@octogonz per your suggestion, I made this a `minor`-level change. This change is technically breaking, but the new behavior is much more predictable.